### DISCUSSION TITLE
upload custom script via WEB GUI error

### DISCUSSION THREAD

**Discussion by jnc27** [netbox-community/netbox#13060] [Q&A] [answered]
Hello,
when trying to upload custom scripts via web gui I get this error:
```
Insufficient write permission to the media root - The configured media root is . Ensure that the user NetBox runs as has access to write files to all locations within this path. 

<class 'PermissionError'>

[Errno 13] Permission denied: '/opt/netbox/netbox/scripts/tst.py'

Python version: 3.9.11
NetBox version: 3.5.4
```

Thank you

**Comment by candlerb** [comment, marked as answer]
You probably need to (re)do the `chown` steps from here: https://docs.netbox.dev/en/stable/installation/3-netbox/#create-the-netbox-system-user

**Reply by jnc27** [reply]
that's it.
Thank you

**Comment by selvigp** [comment]
Hi,
Where do we need to execute the steps mentioned in [https://docs.netbox.dev/en/stable/installation/3-netbox/#create-the-netbox-system-user]. Actually, I have setup my netbox using docker container

**Reply by candlerb** [reply]
The whole point of docker is that you don't need to do any installation at all - you just run the container. So you should completely disregard those instructions.

Instructions for running Netbox as a container using docker-compose are at https://github.com/netbox-community/netbox-docker?tab=readme-ov-file#quickstart
